instrumentation/kamon-logback/src/main/resources/reference.conf

```diff
@@ -13,6 +13,7 @@ kamon.instrumentation.logback {
     trace-id-key = "kamonTraceId"
     span-id-key = "kamonSpanId"
     span-operation-name-key = "kamonSpanName"
+    source-thread-key = "kamonSourceThread"
```

Comment:
Hey @nvollmar! Can you please add a separate comment to this setting stating that this MDC key contains the name of the thread where the log event was created? I think the name is obvious, but having it right there next to the span-related keys makes it look as if the thread name won't be available if there isn't a span, but according to the code this will always be there.

Reply:
Make sense, will add it.